Sources/Extensions/SwiftStdlib/SignedNumericExtensions.swift

```diff
@@ -25,3 +25,22 @@ public extension SignedNumeric {
 	}
 
 }
+
+// MARK: - Methods
+public extension SignedNumeric {
+    
+    /// SwifterSwift: Spelled out representation of a number.
+    ///
+    ///        print((12.32).spelledOutString()) // prints "twelve point three two"
+    ///
+    /// - Parameter locale: Locale, default is .current.
+    /// - Returns: String representation of number spelled in specified locale language. E.g. input 92, output in "en": "ninety-two"
+    public func spelledOutString(locale: Locale = .current) -> String? {
```

Comment:
Please wrap the function with `# if canImport(Foundation)` as well, since `Locale`, `NumberFormatter` and `NSNumber` are not declared in `SwiftStdlib`

Reply:
Added `#if canImport(Foundation)`